src/lib.rs

```diff
@@ -387,6 +388,39 @@
 //! undefined behavior. At some point in the future we may add an attribute to safely place static
 //! variables in this section.
 //!
+//! ## Extra Sections
+//!
+//! Some microcontrollers provide additional memory regions beyond RAM and FLASH.
+//! For example, some STM32 devices provide "CCM" or core-coupled RAM that is
+//! only accessible from the core. In order to access these using
+//! `#[link_section=".foo"]` from your code, you need to modify `memory.x`
```

Comment:
Maybe you could either use the same link section as below (i.e. `#[link_section=".ccmram"]`) to keep the example clearer, or just write something like "In order to access these using the link_section attribute in your code, ...", perhaps with a link to https://doc.rust-lang.org/reference/abi.html#the-link_section-attribute.

Reply:
Good suggestions-- I've updated.